core/spec/helpers/base_helper_spec.rb

```diff
@@ -113,10 +113,10 @@ def link_to_tracking_html(options = {})
 
   # Regression test for #5384
   context "custom image helpers conflict with inproper statements" do
-    let(:product) { mock_model(Spree::Product, :images => [], :variant_images => []) }
+    let(:product) { mock_model(Spree::Product, images: [], variant_images: []) }
     before do
       Spree::Image.class_eval do
-        attachment_definitions[:attachment][:styles].merge!({:foobar => '1x1'})
+        attachment_definitions[:attachment][:styles].merge!({foobar: '1x1'})
```

Comment:
Use `attachment_definitions[:attachment][:styles][:foobar] = '1x1'` instead of `attachment_definitions[:attachment][:styles].merge!({foobar: '1x1'})`.<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.